tests/utils/shopper.js

```diff
@@ -390,6 +390,21 @@ export const shopper = {
 
 			await expect( page.$x( cartItemXPath ) ).resolves.toHaveLength( 1 );
 		},
+
+		applyCouponFromCheckout: async ( couponCode ) => {
+			const couponInputSelector =
+				'#wc-block-components-totals-coupon__input-0';
+			const couponApplyButtonSelector =
+				'.wc-block-components-totals-coupon__button';
+			const couponExpandButtonSelector =
+				'.wc-block-components-totals-coupon button';
+
+			await expect( page ).toClick( couponExpandButtonSelector );
+			await page.waitForSelector( couponInputSelector );
```

Comment:
this shouldn't be needed, expect-puppeteer will wait for element before running an action

Reply:
Great, Thanks! I removed this line of code, and the test is still successful! 👍